__tests__/app.test.js

```diff
@@ -142,6 +142,61 @@ describe("/api/articles/:article_id", () => {
 	});
 });
 
+describe("/api/articles/:article_id/comments", () => {
+	describe("POST: 201", () => {
+		it("should respond with a 201 and the newly created comment object", () => {
+			const article_id = 1;
+			const new_comment = {
+				body: "this is a new comment",
+				author: "lurker",
+				votes: 0,
+			};
+			return request(app)
+				.post(`/api/articles/${article_id}/comments`)
+				.send(new_comment)
+				.expect(201)
+				.then(({ body }) => {
+					const { comment } = body;
+					expect(comment).toHaveProperty(
+						"body",
+						"this is a new comment"
+					);
+					expect(comment).toHaveProperty("author", "lurker");
+					expect(comment).toHaveProperty("votes", 0);
+					expect(comment).toHaveProperty("article_id", article_id);
+					expect(comment).toHaveProperty(
+						"comment_id",
+						expect.any(Number)
+					);
+					expect(comment).toHaveProperty(
+						"created_at",
+						expect.any(String)
+					);
+				});
+		});
+		it("should add the comment to the comments table", () => {
```

Comment:
I would suggest this test is a bit unnecessary. Considering the behaviour of 'RETURNING *' in your SQL query.

Would also reccomend a test where a client has enetered an object with unnecessary keys, and that those keysa re ignored and the post is still succesfull.